scripts/kubectl_wrapper.sh

```diff
@@ -39,7 +39,9 @@ mkdir "${TMPDIR}"
 
 export KUBECONFIG="${TMPDIR}/config"
 
-echo "${CA_CERTIFICATE}" | base64 --decode > "${TMPDIR}/ca_certificate"
+# shellcheck disable=SC1117
+base64 --help | grep "\--decode" && B64_ARG="--decode" || B64_ARG="-d"
```

Comment:
This seems very complex? Why do we need to do all this greping?

Anyways. This is not a testing change = separate PR please.

Reply:
This identifies which flag is used to "decode"; the interface of `base64` differs across platforms.

Reply:
I'd argue that this is indeed a testing change -- in moving from running these tests on the local host (the current state of `master`) to `docker`, it was necessary to change the interface of `base64` as Aaron mentioned above depending on where the test is being run.